app/models/micropost.rb

```diff
@@ -1,3 +1,5 @@
 class Micropost < ActiveRecord::Base
-  validates :content, length: { maximum: 140 }
+  belongs_to :user
+  validates :content, length: { maximum: 140 },
+              presence: true
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.